CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing guidelines
+
+## Pull Request Checklist
+
+Before sending your pull requests, make sure you followed this list.
+
+- Read [contributing guidelines](CONTRIBUTING.md).
+- Ensure you have signed the [Contributor License Agreement (CLA)](https://cla.developers.google.com/).
+- Changes are consistent with the [Coding Style](#coding-style).
+
+## How to become a contributor and submit your own code
+
+### Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+
+Please fill out either the individual or corporate Contributor License Agreement (CLA).
+
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](https://code.google.com/legal/individual-cla-v1.0.html).
+  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](https://code.google.com/legal/corporate-cla-v1.0.html).
+
+Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
+
+### Contributing code
+
+If you have improvements to OpenSesame, send us your pull requests! For those
+just getting started, Github has a
+[how to](https://help.github.com/articles/using-pull-requests/).
+
+OpenSesame team members will be assigned to review your pull requests. Once the
+pull requests are approved by two team members and pass continuous integration
+checks, an OpenSesame member will squash and merge your pul request with the 
+master branch. After approval and merging, the site will be redeployed on a 
+quaterly basis. This is done to ensure that we get a good amount of progress 
```

Comment:
Nit: quarterly
Also, out of curiosity, what made you land on quarterly updates?

Reply:
It felt like a good amount of time. I just looked into it and [this site](https://learn.g2.com/how-often-should-you-update-your-website) says it's good to update websites monthly, I guess I'll change it to monthly.
```suggestion
monthly basis. This is done to ensure that we get a good amount of progress 
```